WooCommerce/WooCommerceTests/ViewRelated/Search/Order/OrderSearchUICommandTests.swift

```diff
@@ -0,0 +1,171 @@
+import XCTest
+import Yosemite
+@testable import WooCommerce
+import Storage
+import Networking
+import protocol WooFoundation.Analytics
+
+final class OrderSearchUICommandTests: XCTestCase {
+    let siteID: Int64 = 12345
+    private var storageManager: MockOrderStatusesStoresManager!
+    private var analyticsProvider: MockAnalyticsProvider!
+    private var analytics: WooAnalytics!
+
+    override func setUpWithError() throws {
+        super.setUp()
+        storageManager = MockOrderStatusesStoresManager()
+        analyticsProvider = MockAnalyticsProvider()
+        analytics = WooAnalytics(analyticsProvider: analyticsProvider)
+    }
+
+    override func tearDownWithError() throws {
+        storageManager = nil
+        analyticsProvider = nil
+        analytics = nil
+        super.tearDown()
+    }
+
+    func testCreateStarterViewControllerReturnsNil() throws {
```

Comment:
```suggestion
    func test_createStarterViewController_returns_nil() {
```
We use snake(ish) case for test names... and I don't think this can throw.

Reply:
Addressed `throws` in commit 162fb9a

Renamed in commit 1b2dc9f